### PR TITLE
Adding Content-sources backend test

### PR DESCRIPTION
**Pull request by mayurilahane**
Add integration test scenario for content-sources backend repo.

### CODE REVIEW

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - name: IQE_FILTER_EXPRESSION
+      value: not (introspection or rbac) # This is the value passed to pytest -k. Default is "" when no filter desired
+#    - name: IQE_REQUIREMENTS
+#      value: # "something,something_else" -- iqe requirements filter. Default is "" when no filter desired
+    - name: IQE_REQUIREMENTS_PRIORITY
+      value: # "something,something_else" -- iqe requirements priority filter. Default is "" when no filter desired
+#    - name: IQE_TEST_IMPORTANCE
+#      value: # "something,something_else" -- iqe test importance filter. Default is "" when no filter desired
+    - name: IQE_CJI_TIMEOUT
+      value: 30m # Timeout value to pass to 'oc wait', should be slightly higher than expected test run time. Default is 30m
```
Comment (akhil-jha) [inline cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:43]: I think you can remove these values.

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#    - name: BONFIRE_COMPONENT_NAME
+#      value: # Name of app-sre "resourceTemplate" in deploy.yaml for this component. If it is the same as the name in Konflux, you don't need to fill this
+    - name: COMPONENT_NAME
+      value: content-sources-backend # Name of your component name in Konflux
+    - name: IQE_PLUGINS
+      value: content-sources # Name of the IQE plugin for this app.
+    - name: IQE_MARKER_EXPRESSION
+      value: api # This is the value passed to pytest -m. Default is ""
+    - name: IQE_FILTER_EXPRESSION
+      value: not (introspection or rbac) # This is the value passed to pytest -k. Default is "" when no filter desired
```
Comment (akhil-jha) [inline cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: Maybe put in quotes?

**Review by akhil-jha — CHANGES_REQUESTED** [review]
I think you missed `EXTRA_DEPLOY_ARGS` from pr_check. Feels like an important thing to me.

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 2 above)
Comment (mayurilahane) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: Yeah i thought so 
But then should other values also be in quotes ?

Comment (akhil-jha) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: If I remember correctly, by default yaml value is strings. So the single word values are fine. Something like `not (introspection or rbac)` I'd keep in quotes as safety thing :P

**Comment by mayurilahane** [comment]
> I think you missed `EXTRA_DEPLOY_ARGS` from pr_check. Feels like an important thing to me.

Yeah, I was in a dilemma about whether to add a new field in this template or not. ?

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 1 above)
Comment (mayurilahane) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:43]: You mean whichever values i dont have i can just remove them ?
or specifically CJI_timeout ?

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 2 above)
Comment (mayurilahane) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: cool, no problem! i will add that then.
Thanks !

**Comment by akhil-jha** [comment]
> > I think you missed `EXTRA_DEPLOY_ARGS` from pr_check. Feels like an important thing to me.
> 
> Yeah, I was in a dilemma about whether to add a new field in this template or not. ?

You should. All such parameters can be added https://github.com/RedHatInsights/bonfire-tekton/blob/main/pipelines/basic.yaml#L115

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 2 above)
Comment (mayurilahane) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: Asking if devs want to simply keep it as "not"

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 1 above)
Comment (akhil-jha) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:43]: yeah, if you don't have a value, let the default take it. So yeah, remove them. It'll look cleaner.

File: cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml (hunk 2 above)
Comment (akhil-jha) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: You'd probably need it. But yeah, I'd say include Stephen into the discussion as well. It's more of a QE thing.

Comment (mayurilahane) [reply cluster/stone-prd-rh01/tenants/rh-hms-tenant/content-sources-backend.yaml:35]: yep just did thanks !

**Comment by akhil-jha** [comment]
LGTM!

**Review by johnbieren — APPROVED** [review]

**Pull request merged**